stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -67,6 +67,7 @@ export
     copyto!,
     copy_transpose!,
     cross,
+    cols,
```

Comment:
I'd call it `columns`. We typically spell things out rather than abbreviating when they aren't too onerous to type.